README.md

```diff
@@ -1318,6 +1318,9 @@ variable. The caching feature hasn't been thoroughly tested and may be thread
 unsafe. If you notice flakiness after enabling it, see the warning below on how
 to workaround it.
 
+**NOTE** Set `CONTAINER_PULL_HTTP_PROXY` and `CONTAINER_PULL_HTTPS_PROXY` to use
```

Comment:
Can we leave a note that the standard HTTP_PROXY and HTTPS_PROXY environment variables are still supported?

Reply:
Done. How is it now?